Makefile

```diff
@@ -129,52 +141,77 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
 
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.1)
-
-KUSTOMIZE = $(shell pwd)/bin/kustomize
-kustomize: ## Download kustomize locally if necessary.
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v3.8.7)
-
-ENVTEST = $(shell pwd)/bin/setup-envtest
-envtest: ## Download envtest-setup locally if necessary.
-	$(call go-get-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
-
-DIGESTPINTOOL = $(shell pwd)/bin/pin-deploy-imgs-in-csv
-digestpintool: ## Download envtest-setup locally if necessary.
-	$(call go-get-tool,$(DIGESTPINTOOL),github.com/opdev/pin-deploy-imgs-in-csv@0.0.1)
```

Comment:
SDO must have pinned digests. You're removing this here, but USE_IMAGE_DIGESTS (added in this PR) is also set to false. Can we enable one or the other, or confirm that this is set at a call point?

Reply:
I'll make sure to set the `USE_IMAGE_DIGESTS=true`